changelog.d/pa-1838.changed

```diff
@@ -0,0 +1,3 @@
+taint-mode: Removed experimental poor-man's support for wrapper functions around
+taint sources. This was an early experiment to make Semgrep inter-procedural, but
+it was later abandoned in favor of DeepSemgrep.
```

Comment:
@IagoAbal do we already have this documented somewhere?

Reply:
If yes, we should remove it. Searching for "wrapper" in our code base, I do not see anything related to taint-mode, unless a different term was used.